website/docs/guides/working-with-guides/core/1-setting-up-examples-on-core-recommended.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Setting up example projects in GitHub Codespaces"
+id: 1-setting-up-examples-on-core-recommended
+slug: working-with-guides/core/1-setting-up-recommended
+description: Learn how to set up Guides example projects on dbt Core in GitHub Codespaces.
+displayText: dbt Core recommended setup
+hoverSnippet: Learn how to set up Guides example projects on dbt Core in GitHub Codespaces.
+---
+
+## Getting started
+
+:::tip Checkout the README
+These instructions along with animated gifs are provided as a walkthrough directly on the repo you'll be using. Feel free to follow the instructions there!
+:::
+
+1. 🔀 **Fork the repo**. Go to the repo specificed as the example project in the guide you're working through, or start from [the base template](https://github.com/dbt-labs/jaffle-shop-template). Click `Use this template` and `Create new repository`.
+2. 🚀 **Open the codespace**. Navigate to that new repository. Where you would normally clone it from the Code button, instead go to the Codespaces tab and click `Create codespace on main`.
+   1. ↔ Depending on your settings, this will **either spin up a browser tab running VSCode** or open the codespace in in a **new window of VSCode** if you have it installed and prefer this. Either option is perfectly fine for this tutorial.
+3. 🍵 **Let the codespace build itself out.** This will take several minutes, grab a cup of tea and come back when it’s setup. It’s building an entire functioning dbt environment and warehouse, stocked with data from a simulated [jaffle](https://craftytoasties.com/what-is-a-jaffle/) shop (a fictional cafe selling food and beverages in several US cities) business for you. You'll be dropped into a fresh terminal tab when the codespace is complete.
```

Comment:
```suggestion
3. 🍵 **Let the codespace build itself out**: This will take several minutes, grab a cup of tea and come back when it’s setup. It’s building an entire functioning dbt environment and warehouse, stocked with data from a simulated [jaffle](https://craftytoasties.com/what-is-a-jaffle/) shop (a fictional cafe selling food and beverages in several US cities) business for you. Once it's done, you'll be directed to a new terminal tab.
```